lambda-src/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "slacknotification",
+  "version": "1.0.0",
+  "description": "Sends a message to a Slack channel",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
also, tests running in CI (travis/drone)

Reply:
yep, the repo now needs a .drone.yml file

Reply:
@chrisns I am planning on doing this as part of CDMP-2337 (pipeline for budget and alert changes) since the drone pipeline for this repo requires a troposphere python virtual env which I'm creating in docker-cdp-ci as part of that other ticket. I trust that approach is ok

Reply:
npm test was added, but I'd like to leave the creation of the .drone.yml file as part of CDMP-2337